src/cascadia/TerminalControl/ScreenInfoUiaProvider.cpp

```diff
@@ -77,10 +82,20 @@ IFACEMETHODIMP ScreenInfoUiaProvider::get_FragmentRoot(_COM_Outptr_result_mayben
     return S_OK;
 }
 
-std::deque<Microsoft::Console::Types::UiaTextRangeBase*> ScreenInfoUiaProvider::GetSelectionRangeUTRs(_In_ Microsoft::Console::Render::IRenderData* pData,
-                                                                                                      _In_ IRawElementProviderSimple* pProvider)
+const COORD ScreenInfoUiaProvider::getFontSize() const
+{
+    return _termControl.GetActualFont().GetSize();
+}
+
+const winrt::Windows::UI::Xaml::Thickness ScreenInfoUiaProvider::GetPadding() const
+{
+    return _termControl.GetPadding();
+}
+
+std::deque<UiaTextRangeBase*> ScreenInfoUiaProvider::GetSelectionRangeUTRs(_In_ IRenderData* pData,

```

Comment:
missing consts on params, here and a bunch below

Reply:
:D you're back, Still catching me for that `const` thing haha